handlers/video_handler.py

```diff
@@ -1,39 +1,15 @@
+from handlers.handler import Handler
+from aiohttp import web, WSMsgType
+import asyncio 
 import cv2
-import asyncio
-from aiohttp import web
-from threading import Thread
-
 from writers.writer import Writer
```

Comment:
Please order the imports like this:
1. Alphabetically all external libraries
2. Newline
3. Alphabetically all internal imports